source/lib-intro.tex

```diff
@@ -3667,13 +3667,13 @@
 A \Cpp{} standard library function shall not directly or indirectly access
 objects\iref{intro.multithread} accessible by threads other than the current thread
 unless the objects are accessed directly or indirectly via the function's arguments,
-including \keyword{this}.
+including the implied object argument\iref{over.match.funcs.general}.
 
 \pnum
 A \Cpp{} standard library function shall not directly or indirectly modify
 objects\iref{intro.multithread} accessible by threads other than the current thread
 unless the objects are accessed directly or indirectly via the function's non-const
-arguments, including \keyword{this}.
+arguments, including the implied object argument\iref{over.match.funcs.general}.
```

Comment:
These two diffs seem more controversial than the rest of the patch. Shouldn't this whole sentence be more like "...via the function's non-const _parameters_, including _the object parameter (if any)_."? We don't mind if evaluating the argument on the _caller's side_ causes an access, and we certainly don't care about the implied object argument (which AFAIK is an artifact of overload resolution on static member functions and doesn't even exist at runtime); we care what the standard library function is physically able to access through its _parameters'_ values.

Reply:
Thanks! Changed as indicated.